docs/tutorial-nodejs.md

```diff
@@ -7,7 +7,7 @@ title: With Node.js
 
 You can work with Accord Project templates directly in JavaScript using Node.js.
 
-Documentation for the API can be found in [Cicero API](https://docs.accordproject.org/docs/cicero-api.html).
+Documentation for the API can be found in [Cicero API](ref-cicero-api).
```

Comment:
This should be `ref-cicero-api.html`

Reply:
Fixed in 9221b7ce9327dad1b1e97cb99ed3608b44baa22c